modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/JournalArticleContentDashboardItem.java

```diff
@@ -273,14 +273,18 @@ public String getUserPortraitURL(HttpServletRequest httpServletRequest) {
 			(ThemeDisplay)httpServletRequest.getAttribute(
 				WebKeys.THEME_DISPLAY);
 
-		try {
-			return _user.getPortraitURL(themeDisplay);
-		}
-		catch (PortalException portalException) {
-			_log.error(portalException, portalException);
+		String userPortraitURL = StringPool.BLANK;
 
-			return StringPool.BLANK;
+		if (_user.getPortraitId() > 0) {
```

Comment:
It's ok, actually there are blocks of code in Portal like this, but why didn't you check the condition at the beginning? this way we may save variable declaration:

```java

@Override
	public String getUserPortraitURL(HttpServletRequest httpServletRequest) {

		if (_user.getPortraitId() <= 0) {
			return StringPool.BLANK;
		}
		
		ThemeDisplay themeDisplay =
			(ThemeDisplay)httpServletRequest.getAttribute(
				WebKeys.THEME_DISPLAY);

		try {
			return _user.getPortraitURL(themeDisplay);
		}
		catch (PortalException portalException) {
			_log.error(portalException, portalException);
			return StringPool.BLANK;
		}
	}
```

Reply:
Right! I'll change it! 👍